package/boot/arm-trusted-firmware-mvebu/patches-a3700-utils/001-version.patch

```diff
@@ -1,13 +1,14 @@
 diff --git a/wtmi/sys_init/Makefile b/wtmi/sys_init/Makefile
```

Comment:
Here we are missing git details like author, description (why this is necessary at all) and if it was send to upstream.

Reply:
The patch already existed, isn't suitable for upstream (it's specific to the openwrt build system) and was just updated, see a04bffebbab51f7728e3bc2c5438f00508c5e708